perl/ANNOTATION.md

```diff
@@ -0,0 +1,781 @@
+# Annotation Fields (Human Assembly hg38 and hg19)
+
+<h2>Annotation Field Description</h2>
+
+##### _Italicized fields_ are custom Bystro fields. All others are sourced as described.
+
+<br/>
+
+#### Summary of the Bystro Annotator
+
+Bystro takes 1+ VCF or SNP (PEMapper/PECaller) files, performs quality control, generates statistics, and outputs an annotated file that is in an easy-to-parse tab-separated format.
+
+Each row in the output file corresponds to a single variant site, and contains a set of annotations that describe the site's genomic context, functional impact, allele frequencies in various populations, and more.
```

Comment:
Lmk if this is incorrect, but doesn't it refer to a single variant? I.e. if you have multiple variants at a single site, they would be on different lines, right? If you had a multi-allelic or indels at the exact same variant site, they would be on multiple lines? If this is the case, would just drop 'site' for clarity.